### PR TITLE
Add taraxa to wagmi default chains

### PR DESCRIPTION
**Pull request by VargaElod23**
## Description

Hey guys! As we're implementing our apps with Wagmi we'd like to add our chains to the default list to make life easier both for us and our community.

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

**Comment by jxom** [comment]
Can we add a changeset for this please?

```
pnpm changeset
```

This can be a `patch`. :)

File: packages/chains/src/taraxa.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: 'Taraxa Mainnet',
+  network: 'taraxa',
+  nativeCurrency: { name: 'Tara', symbol: 'TARA', decimals: 18 },
+  rpcUrls: {
+    default: {
+      http: ['https://rpc.mainnet.taraxa.io'],
+    },
+  },
+  blockExplorers: {
+    default: { name: 'Taraxa Explorer', url: 'https://explorer.mainnet.taraxa.io/' },
```
Comment (jxom) [inline packages/chains/src/taraxa.ts:14]:
```suggestion
    default: { name: 'Taraxa Explorer', url: 'https://explorer.mainnet.taraxa.io' },
```

File: packages/chains/src/taraxaTestnet.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name: 'Taraxa Testnet',
+  network: 'taraxatest',
+  nativeCurrency: { name: 'Tara', symbol: 'TARA', decimals: 18 },
+  rpcUrls: {
+    default: {
+      http: ['https://rpc.testnet.taraxa.io'],
+    },
+  },
+  blockExplorers: {
+    default: { name: 'Taraxa Explorer', url: 'https://explorer.testnet.taraxa.io/' },
```
Comment (jxom) [inline packages/chains/src/taraxaTestnet.ts:14]:
```suggestion
    default: { name: 'Taraxa Explorer', url: 'https://explorer.testnet.taraxa.io' },
```

File: packages/chains/src/taraxaTestnet.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Chain } from './types'
+
+export const taraxaTestnet: Chain = {
+  id: 842,
+  name: 'Taraxa Testnet',
+  network: 'taraxatest',
```
Comment (jxom) [inline packages/chains/src/taraxaTestnet.ts:6]:
```suggestion
  network: 'taraxa-testnet',
```

**Comment by VargaElod23** [comment]
@jxom  All fixed, changeset added

File: .changeset/loud-ladybugs-hang.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/chains': patch
+---
+
+adds taraxa chains
```
Comment (tmm) [inline .changeset/loud-ladybugs-hang.md:5]:
```suggestion
Addesd Taraxa Chains.
```

**Review by tmm — APPROVED** [review]

**Pull request merged**